ports/raspberrypi/boards/arduino_nano_rp2040_connect/mpconfigboard.mk

```diff
@@ -0,0 +1,11 @@
+USB_VID = 0x2E8A
+USB_PID = 0x0003
```

Comment:
Where did this PID come from?

Reply:
I looked it up under Windows device manager